### DISCUSSION TITLE
好东西，说一下我使用中遇到的问题

### DISCUSSION THREAD

**Discussion by Taylor000** [1715173329/UnblockNeteaseMusic#322] [Q&A]
我是直接docker run的，用的PC客户端，一个人用、服务器空载运行；
严格模式下不能播放，能加载但是不能播放；
装了之后的话，一天上传下载要各吃掉10GB左右流量，所以有服务器流量担忧的并且网易云重度用户慎重；
客户端挂代理的情况下，即使你听的歌不需要靠解锁，它还是会加载流量（通过客户端切歌确认）；
只要docker在运行就会一直走流量，不管你有没有在使用（通过代理运行暂停程序确认）；
关于上一条，客户端不设置代理的情况下平均10KB/s，客户端设置代理不在解锁状态平均40KB/s（不包含切歌流量加载）。

总之还是感谢大佬们的维护👍

**Comment by pan93412** [comment]
我自己平時就是用嚴格模式的耶…… 我晚點放我自己的設定參數上來。

**Comment by pan93412** [comment]
```
env ENABLE_LOCAL_VIP=true node app.js -e https://music.163.com -s -p 8080:8081 -o kugou bilibili kuwo
```
